bin/pygrb/pycbc_pygrb_pp_workflow

```diff
@@ -16,185 +16,322 @@
+        # Loudest offsource triggers and onsource trigger html and h5 output files
+        for source_type in ['onsource-trig', 'offsource-trigs']:
+            node.new_output_file_opt(workflow.analysis_time, '.html',
+                                     '--loudest-'+source_type+'-output-file',
+                                     tags=extra_tags+[source_type.upper().replace('-', '_')])
+            node.new_output_file_opt(workflow.analysis_time, '.h5',
+                                     '--loudest-'+source_type+'-h5-output-file',
+                                     tags=extra_tags+[source_type.upper().replace('-', '_')])
+
     # Add job node to the workflow
     workflow += node
 
     return node, node.output_files
 
+# Based on setup_single_det_minifollowups
+def setup_pygrb_minifollowups(workflow, followups_file, ifos,
+                              dax_output, out_dir,
+                              tags=None):
+    """ Create plots that followup the the loudest PyGRB triggers or
+    missed injections from an HDF file.
+    Parameters
+    ----------
+    workflow: pycbc.workflow.Workflow
+        The core workflow instance we are populating
+    followups_file: pycbc.workflow.File
+        The File class holding the triggers/injections.
+    ifos: List of available IFOs to loop over.
+    dax_output: The directory that will contain the dax file.
+    out_dir: path
+        The directory to store minifollowups result plots and files
+    tags: {None, optional}
+        Tags to add to the minifollowups executables
+    """
+
+    logging.info('Entering minifollowups module')
+
+    if not workflow.cp.has_section('workflow-minifollowups'):
+        msg = 'There is no [workflow-minifollowups] section in '
+        msg += 'configuration file'
+        logging.info(msg)
+        logging.info('Leaving minifollowups')
+        return
+
+    tags = [] if tags is None else tags
+    _workflow.makedir(dax_output)
+
+    # Turn the config file into a File class
+    #curr_ifo = single_trig_file.ifo
+    #config_path = os.path.abspath(dax_output + '/' + curr_ifo + \
+    config_path = os.path.abspath(dax_output + '/' + \
+                                   '_'.join(tags) + '_minifollowup.ini')
+    workflow.cp.write(open(config_path, 'w'))
+
+    config_file = resolve_url_to_file(config_path)
+
+    exe = _workflow.Executable(workflow.cp, 'pygrb_minifollowups',
+                               ifos=ifos, out_dir=dax_output, tags=tags)
+
+    #wikifile = curr_ifo + '_'.join(tags) + 'loudest_table.txt'
+    wikifile = '_'.join(tags) + 'loudest_table.txt'
+
+    node = exe.create_node()
+    node.add_input_opt('--config-files', config_file)
+    node.add_input_opt('--followups-file', followups_file)
+    node.add_opt('--wiki-file', wikifile)
+    if tags:
+        node.add_list_opt('--tags', tags)
+    node.new_output_file_opt(workflow.analysis_time, '.dax', '--output-file')
```

Comment:
This would then also update to `--dax-file`. In this case the example is https://github.com/gwastro/pycbc/blob/master/pycbc/workflow/minifollowups.py#L100